cypress/integration/analytics.test.js

```diff
@@ -78,6 +82,7 @@ describe('add_to_cart event', () => {
       cy.itemsInCart(0)
 
       // Add to cart
+      cy.getById('store-card').first().click()
```

Comment:
My suggestion is to go directly to the PDP then click on the buy button.

Reply:
Oh, good call. This is already covered by the test above this one `adds add_to_cart event in the data layer at product description page` and since we removed the possibility of adding to cart via the PLP, it makes sense to remove this one completely.

Done it [here](https://github.com/vtex-sites/base.store/pull/284/commits/119f72545638acaaf414546d6e9429455f002134).